Library/Homebrew/extend/os/mac/formula_cellar_checks.rb

```diff
@@ -48,6 +48,30 @@ def check_openssl_links
     EOS
   end
 
+  def check_accelerate_framework_links
+    return unless @core_tap
+    return unless formula.prefix.directory?
+    return if formula.name == "veclibfort" # veclibfort exists to wrap Accelerate
+
+    keg = Keg.new(formula.prefix)
+    system_accelerate = keg.mach_o_files.select do |obj|
+      dlls = obj.dynamically_linked_libraries
+      # libBLAS and libLAPACK are superseded by functions in OpenBLAS
+      # This can be linked via multiple paths, so use a wildcard match
+      dlls.any? { |dll| %r{^/System/Library/Frameworks/Accelerate.framework/.*/(libBLAS|libLAPACK).dylib$}.match dll }
```

Comment:
If xcode is installed, I have seen formulae link to Frameworks via the Xcode SDK path such as:

~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Accelerate.framework
~~~

so I think `%r{^/System/Li...` won't catch these cases due to the `^`

(this was a source of much consternation to me in https://github.com/osrf/homebrew-simulation/issues/676 and https://github.com/osrf/homebrew-simulation/pull/678, since bottles wouldn't work at runtime without Xcode, but that's a separate issue. )

Reply:
Yikes. Maybe *that* should be an audit too. Thanks for the heads-up.

Reply:
Yeah, probably should be; I'll take a crack at it

Reply:
Now that I look at the fix, the problem for that formula was an Xcode path in a generated cmake file (`SimbodyTargets-release.cmake`). So I'm not sure that any libraries were actually linked to the xcode path